### ISSUE TITLE
Auto config extraction doesn't properly when nth-child() would be needed

### ISSUE THREAD

**Issue by jakopako** [jakopako/goskyr#200]
When within an event there are different fields that have the same selector except different values for nth-child(n) for certain nodes this is not properly detected. Examples for such fields are wrong. eg.

```html
<div class="event">
					              <div class="dt">
              
                <span style="font-size:20px;line-height:20px;">17</span><br />
                <span style="font-size:12px;line-height:14px;">March</span><br />
                <span style="font-size:12px;line-height:14px;">2023</span>
                              </div>
						<div class="concert_thumb">
							<img width="50" height="50" src="http://www.bar-laparenthese.ch/wp-content/uploads/2023/03/primevere-50x50.jpg" class="attachment-concert-list wp-post-image" alt="primevere" title="primevere" />						</div>
              <div class="event_title">
              	<h3><a href="http://www.bar-laparenthese.ch/concerts/primevere-ex-ropoporose-fr-pop-melanconique/" rel="bookmark" title="Primevère (ex-Ropoporose) (FR) &#8211; pop mélanconique">Primevère (ex-Ropoporose) (FR) &#8211; pop mélanconique</a></h3>
              </div><!-- .event_title -->
					
										</div><!-- .event -->
									<div class="event">
					              <div class="dt">
              
                <span style="font-size:20px;line-height:20px;">18</span><br />
                <span style="font-size:12px;line-height:14px;">March</span><br />
                <span style="font-size:12px;line-height:14px;">2023</span>
                              </div>
						<div class="concert_thumb">
							<img width="50" height="50" src="http://www.bar-laparenthese.ch/wp-content/uploads/2023/03/climax-50x50.jpg" class="attachment-concert-list wp-post-image" alt="climax" title="climax" />						</div>
              <div class="event_title">
              	<h3><a href="http://www.bar-laparenthese.ch/concerts/climax-ch-rock-soul-blues/" rel="bookmark" title="Climax (CH) &#8211; rock / soul / blues">Climax (CH) &#8211; rock / soul / blues</a></h3>
              </div><!-- .event_title -->
					
										</div><!-- .event -->
```

This would give as a field-a with examples ["17", "March", "2023", "18", ...] instead of three fields field-a ["17", "18", ..], field-b ["March", "March", ..], field-c ["2023", "2023", ..]

Url: http://www.bar-laparenthese.ch/

**Comment by jakopako** [comment]
More examples:
 - https://www.agharta.cz/

**Comment by jakopako** [comment]
fixed with pr #204
